docs/application/native/guides/machine-learning/overview.md

```diff
@@ -13,7 +13,11 @@ You can use the following machine learning features in your native applications:
 
   You can construct, control, and train a machine learning model on-the-fly. The training can be done using the code or with a set of format that describes the model. These models can be trained on the device locally when needed.
 
-## Related Information
+- [Machine Learning Service](machine-learning-service.md)
+
+  The Machine Learning Service API provides interfaces to store and fetch the pipeline description for AI application developers.
+
+## Related information
 
 - Dependencies
   - Tizen 5.5 and Higher for Mobile
```

Comment:
For other sub pages, "Tizen 5.5 and Higher for Mobile/Wearable" is correct.

However in case of "Machine Learning Service", 
according to added machine-learning-service.md, "Tizen 7.0 and Higher for TV" & "Tizen 7.0 and Higher for IoT" are also supported.
As the Overview page is also covering "Machine Learning Service",
how about adding "Tizen 7.0 and Higher for TV" & "Tizen 7.0 and Higher for IoT" to the overview.md?

Reply:
The dependency information was omitted. I have updated it, so please check again.